### DISCUSSION TITLE
Observations 2022.44.2 with Iphone IOS16.1 - M3 LR 2019 Intel

### DISCUSSION THREAD

**Discussion by brucmod** [tesla-android/issue-tracker#132] [Show and tell]
Thought I would share what I have seen in my experimenting
Pi4 4gb overclocked to 1800MHZ, Over-voltage set to 3 and GPU set to 650 with Low Profile Ice Tower
Auto kit to 

First attempted with unit in center console, Touchscreen Screen Lag made unit unusable
Second Attempt unit in Glove box.  Lag was better, started out usable, about 10 minutes in touchscreen lag made it unusable.
Third attempt with glove box open, things are looking up, touchscreen response much better. (NOT OEM CARPLAY good, but much better)  Navigation audio choppy and inconsistent, but other wise usable. 
Final attempt mounted on dash by Passenger side front window using command strips.  Things are running perfectly. Audio from SIRI /Navigation works great. Touchscreen lag is maybe 95% of OEM, but nothing to complain about.

@mikegapinski has done an amazing job. I am patiently waiting on some of the fixes being implemented around BT and USB tethering of iPhone's to to all for hotspot use for internet access.
Plus the suggestion of having the Pi4 turnoff hotspot when BT is out of range, would also make things cleaner and would force car to connect to LTE for normal connectivity access.

**Comment by mikegapinski** [comment]
What car do you have? Some model 3 cars have the antena on the right mirror on the outside, mine has it up in the pillar above the glovebox, that's why I keep my rig there.

Once things are more optimised it'll run better in terms of bandwidth requirements, navigation audio for instance is RAW PCM right now, not compressed at all

**Reply by brucmod** [reply]
2019 model3 long range AWD

**Comment by adamChrystie** [comment]
@brucmod sorry you are having issues. I'm using 2022.44.2 with CarPlay and speed is near OEM. Siri/NAV audio is rarely choppy. I use an iPhone 13 Pro Max and wireless CarPlay connection. Be sure to set auto kit's audio destination to Bluetooth and not the default setting of box (this is in advanced settings screen I think). Pair Tesla to iPhone via Bluetooth and make the phone a priority device via Tesla bluetooth gui. I also set  autokit's resolution to middle setting and 30FPS and later left it at 60FPS. Highest resolution had more occurrences of choppy audio.

**Reply by brucmod** [reply]
Thanks for the input. Just an FYI, im not having any issues ..  once I moved the PI4 on top of the dash by the passenger window, all my issues went away.   I just wanted to share my methodology with others.  

@mikegapinski does switching autokit settings from 5GHZ too 2.4GHZ have any beneficial effect?

**Reply by adamChrystie** [reply]
I’d try the 2.4ghz setting. Can’t hurt and will probably help. Generally 2.4ghz has a stronger signal and can go through obstructions better. Example, my tesla can connect to my home 2.4ghz from across the street and doesn’t see the 5 ghz Wi-Fi.

**Reply by mikegapinski** [reply]
Use auto kit in 2.4ghz, that is used for the connection between the dongle and your iPhone that happens inside the car. In my testing 5GHz behaves a little bit better
